experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -171,12 +171,36 @@ moisture_flux(::Union{CA.EquilMoistModel, CA.NonEquilMoistModel}, integrator) =
 ρq_tot(::Union{CA.EquilMoistModel, CA.NonEquilMoistModel}, integrator) = integrator.u.c.ρq_tot
 
 # extensions required by the Interfacer
+function Interfacer.get_field(sim::ClimaAtmosSimulation, ::Val{:diffuse_fraction})
+    radiation_model = sim.integrator.p.radiation.rrtmgp_model
+    total_flux_dn = radiation_model.face_sw_flux_dn
+    face_space = axes(sim.integrator.u.f)
+    if radiation_model.radiation_mode isa CA.RRTMGPInterface.GrayRadiation
+        diffuse_fractions = zero(total_flux_dn)
+    else
+        direct_flux_dn = radiation_model.face_sw_direct_flux_dn
+        diffuse_fractions = ((x, y) -> y > zero(y) ? x / y : zero(x)).(total_flux_dn .- direct_flux_dn, total_flux_dn)
```

Comment:
It might be better to only calculate the first half level as that's all you need?